### PR TITLE
batches: temporarily disable namespace selector

### PR DESCRIPTION
**Pull request by courier-new**
Closes https://github.com/sourcegraph/sourcegraph/issues/37135.

Just comments out code related to the namespace selector and adds `TODO` notes with instructions for how to restore. Creating a batch change from the user or org list page will also both temporarily point to the default create page, since the user- and org-specific create pages pre-select a different namespace.

## Test plan

Manually tested that creating a batch change uses the user namespace only. Form has storybook coverage as well and should report a diff to Chromatic.

## App preview:

- [Web](https://sg-web-kr-disable-namespace-selector.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-ddxzglsvie.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by courier-new** [comment]
[Chromatic diff](https://www.chromatic.com/pullrequest?appId=5f0f381c0e50750022dc6bf7&number=37187&view=changes) looks good.

File: client/web/src/enterprise/batches/create/ConfigurationForm.tsx [hunk 1]
```diff
@@ -135,12 +142,14 @@ export const ConfigurationForm: React.FunctionComponent<React.PropsWithChildren<
         <Form className={styles.form} onSubmit={handleCreate}>
             <Container className="mb-4">
                 {error && <ErrorAlert error={error} />}
-                <NamespaceSelector
+                {/* TODO: As we haven't finished implementing support for orgs, we've temporary
```
Comment (eseliger) [inline client/web/src/enterprise/batches/create/ConfigurationForm.tsx:145]: Can we just disable it? Otherwise the UI is so empty 😆

**Review by eseliger — APPROVED** [review]

**Review by Piszmog — APPROVED** [review]

**Review by LawnGnome — APPROVED** [review]
+1 to Erik's comment, otherwise LGTM.

**Review by BolajiOlajide — APPROVED** [review]

File: client/web/src/enterprise/batches/create/ConfigurationForm.tsx (hunk 1 above)
Comment (courier-new) [reply client/web/src/enterprise/batches/create/ConfigurationForm.tsx]: Done!

image

Fixed the mis-matched label text style while I was there. 😄

**Pull request merged**